### ISSUE TITLE
Converter spews out alphanumeric symbols

### ISSUE THREAD

**Issue by taylorshin** [tensorflow/tensorflow#42940]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Mac OS 10.14.6
- TensorFlow installed from (source or binary): source
- TensorFlow version (or github SHA if from source): 2.3.0
- Python 3.8.5

**Command used to run the converter or code if you’re using the Python API**
If possible, please share a link to Colab/Jupyter/any notebook.

```
import tensorflow as tf
from transformers import DistilBertConfig, TFDistilBertForSequenceClassification

model_name = 'distilbert-base-uncased'
config = DistilBertConfig(num_labels=2)
model = TFDistilBertForSequenceClassification.from_pretrained(model_name, config=config)
model.load_weights('tf_model.h5')
converter = tf.lite.TFLiteConverter.from_keras_model(model)
tflite_model = converter.convert()
with tf.io.gfile.GFile('distilbert_sst2.tflite', 'wb') as f:
        f.write(tflite_model)
```

**The output from the converter invocation**
There are 3 sets of debug messages/errors. For the third, I'm not sure if they are error or debug messages but a tsunami of alphanumeric symbols is outputted to the terminal.

1.
```
loc(callsite("tf_distil_bert_for_sequence_classification/distilbert/transformer/layer_._0/ffn/activation_10/Erf"("/Users/taylor/anaconda3/envs/tf2/lib/python3.8/site-packages/tensorflow/python/framework/op_def_library.py":742:0) at callsite("/Users/taylor/anaconda3/envs/tf2/lib/python3.8/site-packages/tensorflow/python/ops/gen_math_ops.py":3295:0 at callsite("/Users/taylor/anaconda3/envs/tf2/lib/python3.8/site-packages/transformers/modeling_tf_distilbert.py":70:0 at callsite("/Users/taylor/anaconda3/envs/tf2/lib/python3.8/site-packages/tensorflow/python/keras/layers/core.py":427:0 at callsite("/Users/taylor/anaconda3/envs/tf2/lib/python3.8/site-packages/tensorflow/python/keras/engine/base_layer.py":985:0 at callsite("/Users/taylor/anaconda3/envs/tf2/lib/python3.8/site-packages/transformers/modeling_tf_distilbert.py":297:0 at callsite("/Users/taylor/anaconda3/envs/tf2/lib/python3.8/site-packages/tensorflow/python/autograph/impl/api.py":302:0 at callsite("/Users/taylor/anaconda3/envs/tf2/lib/python3.8/site-packages/tensorflow/python/keras/engine/base_layer.py":985:0 at callsite("/Users/taylor/anaconda3/envs/tf2/lib/python3.8/site-packages/transformers/modeling_tf_distilbert.py":347:0 at "/Users/taylor/anaconda3/envs/tf2/lib/python3.8/site-packages/tensorflow/python/autograph/impl/api.py":302:0)))))))))): error: 'tf.Erf' op is neither a custom op nor a flex op
```
2.
```
error: failed while converting: 'main': Ops that can be supported by the flex runtime (enabled via setting the -emit-select-tf-ops flag):
	tf.Erf {device = ""}
```
3.
```
FF04BD305693BC0F148BBCC1E52DBC089D96BDE60BE2BC4919B5BCE816F8BC0FB82FBD1A333BBDFFB50BBCB3349...
```

**Also, please include a link to the saved model or GraphDef**

https://drive.google.com/drive/folders/1D5cNxuya7uxFvF7DhmSs1pap-8ATbY08?usp=sharing

**Failure details**
If the conversion is successful, but the generated model is wrong,
state what is wrong:
- Even after 5+ minutes of alphanumeric symbols showing up on the terminal, the `tflite` file is not generated.

**RNN conversion support**
If converting TF RNN to TFLite fused RNN ops, please prefix [RNN] in the title.

**Any other info / logs**

Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by Saduf2019** [comment]
@taylorshin 
I ran the code shared, it is incomplete. Please share complete stand alone code if possible or share a  colab gist with the error faced.
With respect tot he error "Ops that can be supported by the flex runtime" can you refer tot his issue and let us know #42184

**Comment by taylorshin** [comment]
I updated my code. The `in_file` should have been `tf_model.h5`, which is the weights file included in the Google Drive link. I also added the import statements so the code should be able to run now.

**Comment by abattery** [comment]
The erf operator currently are not supported by TFLite builtin operator set. You need to enable Select TF op feature to convert your model. Please refer to the following documents:
https://www.tensorflow.org/lite/guide/ops_select
https://www.tensorflow.org/lite/guide/reduce_binary_size

**Comment by Saduf2019** [comment]
@taylorshin 
Please update as per above comment, and feel free to move the issue to closed status if resolved.

**Comment by taylorshin** [comment]
Enabling the Select TF op feature resulted in the model converting without spewing alphanumeric symbols. However, the converted TFLite model has a fixed input size of `3 x 5`. Could that have been set by the TFLite conversion process?

![Screen Shot 2020-09-04 at 3 00 00 PM](https://user-images.githubusercontent.com/10328411/92506626-925cfc80-f1ba-11ea-81fe-33edb371c9a2.png)

**Comment by abattery** [comment]
TFLite converter uses the same shape information from the original TF model.

**Comment by taylorshin** [comment]
So I used a library called `ktrain` to fine-tune DistilBert on SST2 and got a weights h5 file which is included in the Google Drive link in my original post. All I do to convert that into a TFLite model is load a pretrained `TFDistilBertForSequenceClassification` from HuggingFace and load the weights onto the model. According to the author of the library, `ktrain` is most likely not the reason for the fixed input size because all it does is set the output shape, metrics, and loss functions of existing HuggingFace models.

**Comment by abattery** [comment]
You can override the input shape with V1 converter API. Please take a look at the input_shapes argument in the following document.

https://www.tensorflow.org/api_docs/python/tf/compat/v1/lite/TFLiteConverter#from_keras_model_file

**Comment by sushreebarsa** [comment]
@taylorshin It looks like you are using an older Version of Tensorflow. Many bugs have been fixed in the latest version. Could you please execute your code using Latest Version 2.5 and let us know if the issue still persists? Please have a look at this[ link](https://www.tensorflow.org/api_docs/python/tf/compat/v1/lite/TFLiteConverter#from_keras_model_file) and let us know if it helps ?Thanks!
